lingua_franca/lang/common_data_sl.py

```diff
@@ -0,0 +1,173 @@
+#
+# Copyright 2017 Mycroft AI Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from collections import OrderedDict
+
+
+_ARTICLES_SL = {}
```

Comment:
if articles exist they should be here, even if not used? any reason they are not replaced in normalize? eg, in portuguese and spanish articles are important and can not be removed in normalize, but they should still be present here

Reply:
Slovenian has no articles. Most of the information available from articles in other languages (e.g. declensions, gender, definiteness, indefiniteness) can be determined from the word itself or accompanying adjectives, so I think nothing should be listed here.